$NetBSD: patch-compat_Makefile.am,v 1.2 2019/07/02 20:52:13 nia Exp $

libcompat must be prebuilt with PIC properties in order to be linked
into PIC libraries.

--- compat/Makefile.am.orig	2018-03-14 22:07:38.000000000 +0000
+++ compat/Makefile.am
@@ -1,7 +1,7 @@
-pkglib_LTLIBRARIES = libcompat.la
+noinst_LTLIBRARIES = libcompat.la
 libcompat_la_SOURCES = dummy.c
 libcompat_la_LIBADD = $(LTLIBOBJS) $(LIBRT) $(LIBM)
-libcompat_la_LDFLAGS = -no-undefined -static
+libcompat_la_LDFLAGS = -no-undefined
 
 BUILT_SOURCES = dummy.c
 CLEANFILES = dummy.c
